src/model/channel/message.rs

```diff
@@ -72,6 +72,15 @@ pub struct Message {
     /// Array of [`Role`]s' Ids mentioned in the message.
     pub mention_roles: Vec<RoleId>,
     /// Channels specifically mentioned in this message.
+    ///
+    /// **Note**:
+    /// Not all channel mentions in a message will appear in mention_channels.
```

Comment:
```suggestion
    /// Not all channel mentions in a message will appear in `mention_channels`.
```
We refer to fields and other Rust items by `ticks`.